uh-groupings-ui-overrides.skeleton.properties

```diff
@@ -1,4 +1,4 @@
-# Create a directory in your home directory called ".<username for your computer>-conf"
+# Create a directory in your home directory called ".<uid for your computer>-conf"
```

Comment:
Keep this as `username` since the comment is talking about your computer's username, not a subject.